README.md

```diff
@@ -1,25 +1,38 @@
-# Bitmaker Cloud CLI
+<h1 align="center"> Bitmaker Cloud CLI </h1>
 
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
+Bitmaker Cloud CLI is the command line client to interact with Bitmaker Cloud API. Allows the user to perform the following actions:
+- Link a Scrapy project with a project in Bitmaker Cloud.
+- Create projects, jobs and cronjobs in Bitmaker Cloud.
```

Comment:
Currently, we can't create cronjobs via the client 😅

Reply:
Sure... I removed